typescript/lib/bulk_command_execution/v0_bulk_command_api/v0_bulk_commands.ts

```diff
@@ -0,0 +1,260 @@
+import { LoadLambdaArgs } from "../../../kurtosis_core_rpc_api_bindings/api_container_service_pb";
+import { newEmptyExecCommandArgs, newEmptyExecuteBulkCommandsArgs, newEmptyExecuteLambdaArgs, newEmptyGenerateFileArgs, newEmptyLoadLambdaArgs, newEmptyLoadStaticFilesArgs, newEmptyRegisterServiceArgs, newEmptyRemoveServiceArgs, newEmptyRepartitionArgs, newEmptyStartServiceArgs, newEmptyWaitForEndpointAvailabilityArgs } from "../../../lib/constructor_calls";
+import { ExecCommandArgs, ExecuteBulkCommandsArgs, ExecuteLambdaArgs, GenerateFilesArgs, LoadStaticFilesArgs, RegisterServiceArgs, RemoveServiceArgs, RepartitionArgs, StartServiceArgs, WaitForEndpointAvailabilityArgs } from "../../../kurtosis_core_rpc_api_bindings/api_container_service_pb";
+import { V0CommandTypeVisitor, V0CommandType, acceptVisitor } from "./v0_command_types";
+import { ok, err, Result } from "neverthrow";
+import * as jspb from "google-protobuf";
+
+// ====================================================================================================
+//                                   Command Arg Deserialization Visitor
+// ====================================================================================================
+
+// Visitor that will be used to deserialize command args into
```

Comment:
Haha oops, looks like I forgot to complete the sentence :P No need to change though on your side; I'll fix it one day